### ISSUE TITLE
(Checkbox, Radio button) Clicking on list item/text/label does not check-uncheck these

### ISSUE THREAD

**Issue by ubill** [OnsenUI/OnsenUI#2351]
```
[Core]
    onsenui 2.9.1

[Framework]
    vue "2.5.13"

[Framework binding]
    vue-onsenui 2.5.0

[Additional libraries]
    jQuery 3.1.1

[Platform]
    Desktop - Windows 10

[Browser]
    Desktop - Chrome 63.0.3239.132 (Official Build) (64-bit)
```

__Encountered problem__
Copy-pasting the exact code either from the playground, into my own Vue project,
[https://onsen.io/playground/?framework=vue&category=reference&module=input](https://onsen.io/playground/?framework=vue&category=reference&module=input)
or from the docs
[https://onsen.io/v2/api/vue/v-ons-checkbox.html](https://onsen.io/v2/api/vue/v-ons-checkbox.html)

, the checkbox and the radio button can only be toggled by clicking exactly on the (tick mark) or (checkbox), and cannot be toggled by clicking on the entire tappable list item.

__How to reproduce__

I am using a .vue file as a component, and my template is thus:

```
<script>
import viewmodel from '../../ccr/viewmodel';
window.viewmodel = viewmodel;

export default {
  data: function() {
    return {
      name: '',
      vegetables: ['Apples', 'Bananas', 'Oranges'],
      selectedVegetable: 'Bananas',
      colors: ['Red', 'Green', 'Blue'],
      checkedColors: ['Green', 'Blue'],

      viewmodel
    };
  }
};
</script>

<style scoped lang="scss">

</style>

<template>


  <v-ons-page>
    <v-ons-toolbar>
      <div class="center">Form input</div>
    </v-ons-toolbar>

    <p>
      <v-ons-search-input placeholder="Search something"></v-ons-search-input>
    </p>

    <v-ons-list>
      <v-ons-list-header>
        Text input
      </v-ons-list-header>
      <v-ons-list-item>
        <div class="center">
          <v-ons-input placeholder="Input your name" float
            v-model="name"
          >
          </v-ons-input>
        </div>
      </v-ons-list-item>
      <v-ons-list-item>
        <div class="right">
          Hello {{ name || 'anonymous' }}!
        </div>
      </v-ons-list-item>

      <v-ons-list-header>
        Radio buttons
      </v-ons-list-header>
      <v-ons-list-item v-for="(vegetable, $index) in vegetables" :key="vegetable" tappable>
        <label class="left">
          <v-ons-radio
            :input-id="'radio-' + $index"
            :value="vegetable"
            v-model=" selectedVegetable"
          >
          </v-ons-radio>
        </label>
        <label :for="'radio-' + $index" class="center">
          {{ vegetable }}
        </label>
      </v-ons-list-item>
      <v-ons-list-item>
        <div class="center">
          I love {{ selectedVegetable }}!
        </div>
      </v-ons-list-item>

      <v-ons-list-header>
        Checkboxes - {{checkedColors}}
      </v-ons-list-header>
      <v-ons-list-item v-for="(color, $index) in colors" :key="color" tappable>
        <label class="left">
          <v-ons-checkbox
            :input-id="'checkbox-' + $index"
            :value="color"
            v-model="checkedColors"
          >
          </v-ons-checkbox>
        </label>
        <label class="center" :for="'checkbox-' + $index">
          {{ color }}
        </label>
      </v-ons-list-item>



      <v-ons-list-item>
        <label class="left">
          <v-ons-checkbox input-id="inner-checkbox"></v-ons-checkbox>
        </label>
        <label class="center" for="inner-checkbox">
          Clicking this text triggers the checkbox
        </label>
      </v-ons-list-item>




    </v-ons-list>
  </v-ons-page>



</template>
```

As you can see, there is nothing that is out of the ordinary.
Initially I tried copy-pasting directly the code from those two URLs (they are identical anyway) and it didn't work in the for loop, 
so I tried putting a single list item at the bottom, it didn't work too.

One difference I see in the produced HTML elements when using Chrome devtools is an additional data attr for my code (pasted below later), but from the playground or the API docs, this data attr is not present. Could this have any effect?

```
# My produced elem

<ons-list-item data-v-740754b9="" tappable="" class="list-item"><label data-v-740754b9="" class="left list-item__left"><ons-checkbox data-v-740754b9="" input-id="checkbox-0" value="Red" class="checkbox">
      <input type="checkbox" class="checkbox__input" id="checkbox-0" value="Red">
      <span class="checkbox__checkmark"></span>
    </ons-checkbox></label> <label data-v-740754b9="" for="checkbox-0" class="center list-item__center">
        Red
      </label></ons-list-item>
```

```
# The playground/docs produced elem

<ons-list-item tappable="" class="list-item"><label class="left list-item__left"><ons-checkbox input-id="checkbox-0" value="Red" class="checkbox">
      <input type="checkbox" class="checkbox__input" id="checkbox-0" value="Red">
      <span class="checkbox__checkmark"></span>
    </ons-checkbox></label> <label for="checkbox-0" class="center list-item__center">
          Red
        </label></ons-list-item>
```

As you can see there is this data `data-v-740754b9=""` attr present which is the difference.

**Comment by ubill** [comment]
An additional comment is that I googled and looked at past issues, nothing helped.

**Comment by frandiox** [comment]
@ubill Hi! No, the `data-v` attribute is added by Vue for scoped css or something else. The list item is actually unrelated to the inputs. The click is propagated to the inputs thanks to the native `<label>` element and its `for` attribute. Maybe you can check if it works with native HTML elments, something like:

```
<label for="checkbox-0">click here</label><input type="checkbox" id="checkbox-0">
```

**Comment by ubill** [comment]
Great prompt reply @frandiox , I inspected the entire page and I found that elsewhere, I had elements of the same `id` (not visible, behind in the page stack) and those were getting the `label for='id'` update.

I removed the elems with the same `id`s so that only one set is on the page (as it should be), then the list-item checkbox/radio widgets & labels started working as expected. False alarm, then.

Cheers! And I will close this issue now.
